routers/api/v1/api.go

```diff
@@ -510,6 +517,10 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Get("/swagger", misc.Swagger) // Render V1 by default
 	}
 
+	m.Group("/", func() {
+		m.Post("/graphql", gql.GraphQL)
+	}, securityHeaders(), context.APIContexter(), sudo())
+
```

Comment:
suggest add this entry to ``/v1`` path also.

Reply:
Most graphql stuff I read seemed to recommend avoiding versioning if possible, but like most decisions it is debatable, for example: https://graphql.org/learn/best-practices/#versioning